test/ibmq/test_ibmq_job.py

```diff
@@ -296,6 +296,34 @@ def test_retrieve_jobs_status(self, provider):
         for job in job_list:
             self.assertTrue(job.status() is JobStatus.DONE)
 
+    @skip('Enable when api fixes job statuses')
+    @requires_provider
+    def test_retrieve_jobs_queued(self, provider):
+        """Test retrieving jobs that are queued."""
+        backend = max([b for b in provider.backends(simulator=False) if b.status().operational],
+                      key=lambda b: b.status().pending_jobs)
```

Comment:
Since the main purpose of adding this function is for testing purposes, would placing it into `ibmq.utils.utils` be best? Another possible alternative was to place it in `ibmq.__init__` along with `least_busy`, but I did not add it there since the function was added for testing and not users per se...

Reply:
Oh I meant in a new `test/utils.py`, not in the source code package.